packages/pn-commons/src/components/Layout/Layout.tsx

```diff
@@ -1,9 +1,16 @@
 import { ReactNode } from 'react';
 import { Stack } from '@mui/material';
-import { ProductEntity, JwtUser, PartyEntity, UserAction } from '@pagopa/mui-italia';
+import { ProductEntity, JwtUser, PartyEntity, UserAction, Footer } from '@pagopa/mui-italia';
```

Comment:
Non utilizzare direttamente il Footer di mui ma quello in pn-commons come fatto inizialmente.
Modificare in caso quello per adattarlo alle esigenze